### PR TITLE
plat-371 MDS-agency mocha to jest conversion

### PR DESCRIPTION
**Pull request by nessur**
## 📚 Purpose
*[convert all tests from mocha to test]*

## 👌 Resolves:
- [x] ✨ PLAT-371 death to mocha (mds-agency part)

## 📦 Impacts:
*[mds-agency]*

## ✅ PR Checklist
- [x] integration-tests
- [x] request-handlers
- [x] sandbox-admin
- [x] taxi-tests

### CODE REVIEW

File: packages/mds-agency/tests/request-handlers.spec.ts [hunk 1]
```diff
@@ -150,38 +149,13 @@ describe('Agency API request handlers', () => {
...
       await registerVehicle({ body } as AgencyApiRegisterVehicleRequest, res)
-      assert.equal(statusHandler.calledWith(201), true)
-      assert.equal(sendHandler.called, true)
+      expect(statusHandler.calledWith(201)).toBeTruthy()
+      expect(sendHandler.called).toBeTruthy()
       Sinon.restore()
     })
   })
 
   describe('Get vehicle by id', () => {
```
Comment (PeterSP) [inline packages/mds-agency/tests/request-handlers.spec.ts:158]: Why cut this test?

File: packages/mds-agency/tests/sandbox-admin-request-handlers.spec.ts [hunk 2]
```diff
@@ -1,36 +1,24 @@
...
     it('Fails to get cache info', async () => {
       const res: AgencyApiResponse = {} as AgencyApiResponse
-      const sendHandler = Sinon.fake.returns('asdf')
-      const statusHandler = Sinon.fake.returns({
-        send: sendHandler
-      } as any)
-      res.status = statusHandler
-      Sinon.replace(cache, 'info', Sinon.fake.rejects('it-failed'))
+      res.status = jest.fn().mockImplementationOnce(() => ({ send: jest.fn().mockImplementation(() => 'okay') }))
+      jest.spyOn(cache, 'info').mockImplementationOnce(async () => await Promise.reject({ okay: 'there' }))
```
Comment (PeterSP) [inline packages/mds-agency/tests/sandbox-admin-request-handlers.spec.ts:18]: Neat

**Review by PeterSP — COMMENTED** [review]
Other than the cut test I don't understand, looks great!

File: packages/mds-agency/tests/request-handlers.spec.ts (hunk 1 above)
Comment (nessur) [reply packages/mds-agency/tests/request-handlers.spec.ts:158]: I talked with @avatarneil and he says this test is worthless, and was causing timeouts, unclosed handles.

**Review by PeterSP — APPROVED** [review]

**Pull request merged**